core/templates/pages/skill-editor-page/editor-tab/skill-concept-card-editor/worked-example-editor.directive.ts

```diff
@@ -61,9 +61,9 @@ angular.module('oppia').directive('workedExampleEditor', [
             }
             var contentHasChanged = ((
               $scope.workedExampleQuestionMemento !==
-              $scope.container.workedExampleQuestionHtml) ||
-              ($scope.workedExampleExplanationMemento !==
-              $scope.container.workedExampleExplanationHtml)
+              $scope.container.workedExampleQuestionHtml) || (
+              $scope.workedExampleExplanationMemento !==
```

Comment:
is there any issue with the old code? can't we break after || if it's already in parenthesis (line 62)?